packages/jest-circus/src/__tests__/__snapshots__/hooks.test.js.snap

```diff
@@ -74,3 +83,33 @@ run_finish
 
 unhandledErrors: 0"
 `;
+
+exports[`multiple before each hooks in one describe are executed in the right order 1`] = `
+"start_describe_definition: describe 1
+add_hook: beforeEach
+add_hook: beforeEach
+start_describe_definition: 2nd level describe
+add_test: test
+finish_describe_definition: 2nd level describe
+finish_describe_definition: describe 1
+run_start
+run_describe_start: ROOT_DESCRIBE_BLOCK
+run_describe_start: describe 1
+run_describe_start: 2nd level describe
+test_start: test
+hook_start: beforeEach
+before each 1
```

Comment:
before the fix this would print `before each 2` and the next one (line 104) would be `before each 1`

Reply:
I like these snapshots, really easy to review